### PR TITLE
feat(prompts):Adding CSV and Pros-Cons prompts

### PR DESCRIPTION
**Pull request by cenksezgin**
- Adding CSV for extracting data
- Adding Pros and Cons in a Table Format

### CODE REVIEW

File: src/prompts/prompts.toml [hunk 1]
```diff
@@ -156,6 +156,20 @@ prompt = '''
...
+prompt = '''
+Give me in a CSV format: 
+'''
+
+
+[pros-cons]
+name = "Pros and Cons Table"
+description = "Pros and Cons Table"
+prompt = '''
+Give me Pros and Cons in a table for :
```
Comment (briansunter) [inline src/prompts/prompts.toml:171]: Could you remove the space between for and : ?

**Review by cenksezgin — COMMENTED** [review]
made the change you requested

**Comment by briansunter** [comment]
Looks good! Thanks for the contribution.

**Pull request merged**